h2o-genmodel/src/main/java/hex/genmodel/tools/MojoPrinter.java

```diff
@@ -0,0 +1,6 @@
+package hex.genmodel.tools;
+
+public interface MojoPrinter {
```

Comment:
why do we need this ifc? I think there is couple of possible better solutions
a)  make JgraphtPrintMojo extend printMojo 
b) refactor PringMojo to be extendable via service interface and maybe also extract JSON into separate service

Reply:
did SPI approach, JgraphtPrintMojo extends PrintMojo and MojoPrinter is used as service interface